File: 30_days_of_code/operators/cpp/operators.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iomanip>

using namespace std;

int main() {
  double meal_cost;
  int tip_percent, tax_percent;

  double tip, tax;

  cin >> meal_cost;
  cin >> tip_percent;
  cin >> tax_percent;

  tip = meal_cost * tip_percent / 100;
  tax = meal_cost * tax_percent / 100;

  cout << "The total meal cost is " << (int)round(meal_cost + tip + tax) << " dollars." << endl;

  return 0;
}

File: 30_days_of_code/operators/elixir/operators.ex
case Float.parse(IO.gets("")) do
  {meal_cost, _} ->
    case Integer.parse(IO.gets("")) do
      {tip_percent, _} ->
        case Integer.parse(IO.gets("")) do
          {tax_percent, _} ->
            IO.puts "The total meal cost is "
            <> Integer.to_string(round(meal_cost +
              meal_cost * tip_percent / 100 +
              meal_cost * tax_percent / 100))
            <> " dollars."
        end
    end
end
